Dynatrace/dynatrace-operator
# Fix nil-pointer in conflicting namespace validation

## Description

Currently it could happen that a nilpointer exception happened in the webhook, if two dynakubes with conflicting namespace selectors were applied.

## How can this be tested?

Apply a Dynakube with a namespace selector and applicationMonitoring

Apply a Dynakube with classicFullStack and metadataEnrichment enabled for all namespaces

Check if no nilpointer is logged
